### PR TITLE
Wild fire added outputs to the File command

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31379

## Description
added outputs to the File command

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py [hunk 1]
```diff
@@ -527,6 +539,22 @@ def create_file_report(file_hash: str, reports, file_info, format_: str = 'xml',
                     if '-text' in report["evidence"]["file"]["entry"]:
                         evidence_text.append(report["evidence"]["file"]["entry"]["-text"])
 
+        if 'elf_info' in report and report["elf_info"]:
+            if 'Domains' in report["elf_info"]:
```
Comment (ShahafBenYakir) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py:543]:
```suggestion
            if 'Domains' in report["elf_info"] and report["elf_info"]["Domains"]:
```

**Review by ShahafBenYakir — COMMENTED** [review]
1. Add RN
2. Add to integration documentation.
2. For all the drill-downs in dictionaries why not use `demisto.get`?
3. I don't see WF in the google sheet on the `FeedRelatedIndicators` item. Did Yael talk to you privately asking you to add that?

File: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/1_2_4.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks WildFire v2
+- - Added outputs to the **File** command.
```
Comment (ShahafBenYakir) [inline Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/1_2_4.md:4]:
```suggestion
- - Added new outputs to the **File** command.
```

Comment (ShahafBenYakir) [reply Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/1_2_4.md:4]: Also need to add a message about the docker image bump.

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml [hunk 3]
```diff
@@ -110,6 +110,21 @@ script:
     - contextPath: InfoFile.Type
       description: The report file type.
       type: string
+    - contextPath: File.FeedRelatedIndicators.value
+      description: Indicators that are associated with the File
```
Comment (roysagi) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml:114]:
```suggestion
      description: Indicators that are associated with the File.
```

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml [hunk 4]
```diff
@@ -110,6 +110,21 @@ script:
...
+      type: String
+    - contextPath: File.FeedRelatedIndicators.type
+      description: The type of the indicators that are associated with the File
```
Comment (roysagi) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml:117]:
```suggestion
      description: The type of the indicators that are associated with the File.
```

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml [hunk 5]
```diff
@@ -110,6 +110,21 @@ script:
...
+      type: String
+    - contextPath: File.Tags
+      description: Tags that are associated with the File
```
Comment (roysagi) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml:120]:
```suggestion
      description: Tags that are associated with the File.
```

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml [hunk 6]
```diff
@@ -110,6 +110,21 @@ script:
...
+      type: String
+    - contextPath: File.Behavior.details
+      description: File behavior details
```
Comment (roysagi) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml:123]:
```suggestion
      description: File behavior details.
```

File: Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml [hunk 7]
```diff
@@ -110,6 +110,21 @@ script:
...
+      type: String
+    - contextPath: File.Behavior.action
+      description: File behavior action
```
Comment (roysagi) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml:126]:
```suggestion
      description: File behavior action.
```

**Pull request merged**